delivery/media/dash/lib/media-delivery-dash-parser.js

```diff
@@ -0,0 +1,2006 @@
+import { logger } from "log";
+
+const DashConstants = new class {
+    constructor() {
+        this.ACCESSIBILITY = "Accessibility", this.ADAPTATION_SET = "AdaptationSet", this.ASSET_IDENTIFIER = "AssetIdentifier", 
```

Comment:
treeshaker is unable to remove the unused constants. will check more on this.

Reply:
While browsing I found this doc : https://github.com/rollup/rollup/issues/3989 , which says rollup do not have "lazy binding" for certain scenarios . since rollup wouldn't know when will we be using these constants in our code, rollup doesn't remove any of them.